patches/minecraft/net/minecraft/world/level/Level.java.patch

```diff
@@ -206,6 +207,9 @@
                      blockstate = this.getBlockState(blockpos);
 -                    if (blockstate.is(Blocks.COMPARATOR)) {
 +                    if (blockstate.getWeakChanges(this, blockpos)) {
++                        // Forge: Use IForgeBlockState#onNeighborChange, the default impl redirects to Level#neighborChanged
++                        blockstate.onNeighborChange(this, blockpos, p_46718_);
++                        if (false)
                          this.neighborChanged(blockstate, blockpos, p_46719_, p_46718_, false);
```

Comment:
Just delete the line.. no need for if(false) we're deleting a line and replacing it with like we do in modern versions.

Reply:
Ok, I wasn't sure since `if (false)` is used in other places. Noted that full-on replacements to just delete the original line.